fvtest/gctest/GCConfigTest.cpp

```diff
@@ -300,18 +307,8 @@ GCConfigTest::allocateHelper(const char *objName, uintptr_t size)
 	objEntry.name = objName;
 	objEntry.objPtr = NULL;
 
-	uint8_t objectAllocationModelSpace[sizeof(MM_ObjectAllocationModel)];
-	MM_ObjectAllocationModel *noGc = new(objectAllocationModelSpace)
-			MM_ObjectAllocationModel(env, size, MM_ObjectAllocationModel::selectObjectAllocationFlags(false, false, false, true));
-	objEntry.objPtr = OMR_GC_AllocateObject(exampleVM->_omrVMThread, noGc);
-
-	if (NULL == objEntry.objPtr) {
-		gcTestEnv->log("No free memory to allocate %s of size 0x%llx, GC start.\n", objName, size);
-		MM_ObjectAllocationModel *withGc = new(objectAllocationModelSpace)
-				MM_ObjectAllocationModel(env, size, MM_ObjectAllocationModel::selectObjectAllocationFlags(false, false, false, false));
```

Comment:
Can you explain this change in more detail?  It looks like we are removing part of the test that verifies we can allocate without a GC happening if we pass the right flags.

Reply:
You are correct, I will revert this changes and retest gctest. Thanks for spotting this.